java-generator/core/src/main/java/io/fabric8/java/generator/nodes/AbstractJSONSchema2Pojo.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.java.generator.nodes;
+
+import static io.fabric8.java.generator.nodes.Keywords.JAVA_KEYWORDS;
+
+import com.github.javaparser.ast.CompilationUnit;
+import io.fabric8.java.generator.exceptions.JavaGeneratorException;
+import io.fabric8.kubernetes.api.model.apiextensions.v1.JSONSchemaProps;
+import java.util.function.Function;
+
+public abstract class AbstractJSONSchema2Pojo {
+
+    static final String BOOLEAN_CRD_TYPE = "boolean";
+    static final String INTEGER_CRD_TYPE = "integer";
+    static final String INT32_CRD_TYPE = "int32";
+    static final String INT64_CRD_TYPE = "int64";
+    static final String NUMBER_CRD_TYPE = "number";
+    static final String FLOAT_CRD_TYPE = "float";
+    static final String DOUBLE_CRD_TYPE = "double";
+    static final String STRING_CRD_TYPE = "string";
+    static final String OBJECT_CRD_TYPE = "object";
+    static final String ARRAY_CRD_TYPE = "array";
+
+    public abstract String getType();
+
+    public abstract GeneratorResult generateJava(CompilationUnit cu);
+
+    /** Takes a random string and manipulate it to be a valid Java identifier */
+    public static String sanitizeString(String str) {
+        String sanitized = "";
+        if (JAVA_KEYWORDS.stream().anyMatch(s -> s.equals(str))) {
```

Comment:
Why not `contains()`? (maybe also convert the JAVA_KEYWORDS to a Set)

Reply:
done 👍